src/graph/CreateEdgeIndexExecutor.cpp

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "graph/CreateEdgeIndexExecutor.h"
+
+namespace nebula {
+namespace graph {
+
+CreateEdgeIndexExecutor::CreateEdgeIndexExecutor(Sentence *sentence,
+                                                 ExecutionContext *ectx) : Executor(ectx) {
+    sentence_ = static_cast<CreateEdgeIndexSentence*>(sentence);
+}
+
+Status CreateEdgeIndexExecutor::prepare() {
+    return Status::OK();
+}
+
+void CreateEdgeIndexExecutor::execute() {
+    auto status = checkIfGraphSpaceChosen();
+    if (!status.ok()) {
+        DCHECK(onError_);
+        onError_(std::move(status));
+        return;
+    }
+
+    auto *mc = ectx()->getMetaClient();
+    auto *name = sentence_->indexName();
+    auto spaceId = ectx()->rctx()->session()->space();
+
+    std::map<std::string, std::vector<std::string>> indexProperties;
+    indexProperties.emplace(*(sentence_->edgeName()), std::move(sentence_->names()));
```

Comment:
Currently, we only support one tag or edgetype . So there is no need to use map？WDYT？

Reply:
emmm... this is a long story. Long long ago, we design the index will across multi tag or edge.